### ISSUE TITLE
VS Code analyzer reports error even when server starts

### ISSUE THREAD

**Issue by ErikEJ** [ErikEJ/SqlServer.Rules#694]
This is the output:

```text
Starting analyzer server: dotnet dnx ErikEJ.DacFX.TSQLAnalyzer.Cli --yes -- --server-mode
TSQLAnalyzer server mode started. Ready for requests.
Analyzer error for c:\dev\GitHub\SqlServer.Rules\sqlprojects\AdhocScripts\AdhocDml.sql: Path is required for analyze command
```
So the server starts, but then an error is reported
